apps/chat/src/constants/client-data.ts

```diff
@@ -0,0 +1,2 @@
+export const CLIENTDATA_PATH = 'clientdata';
+export const INSTALLED_DEPLOYMENTS = 'installed_deployments.json';
```

Comment:
I think these variables are destined to be in the config, what do you think about that @IlyaBondar?

Reply:
no, it shouldn't be configured